### ISSUE TITLE
Example Livebooks have deps that error

### ISSUE THREAD

**Issue by GenericJam** [elixir-nx/axon#493]
Pretty much all the Livebooks from the Axon guides have deps that don't work out of the box.  It's 4/4 so far.

To rectify this I've been deleting them and adding them in manually which grabs the latest.

The error I'm getting:

```
Unchecked dependencies for environment dev:
* nx (https://github.com/elixir-nx/nx.git)
  the dependency does not match the requirement "~> 0.3.0", got "0.5.1"


** (Mix.Error) Can't continue due to errors on dependencies
    (mix 1.14.2) lib/mix.ex:513: Mix.raise/2
    (mix 1.14.2) lib/mix/tasks/deps.loadpaths.ex:43: Mix.Tasks.Deps.Loadpaths.run/1
    (mix 1.14.2) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.2) lib/mix.ex:782: anonymous fn/5 in Mix.install/2
    (elixir 1.14.2) lib/file.ex:1607: File.cd!/2
    (mix 1.14.2) lib/mix.ex:749: Mix.install/2
    (stdlib 4.2) erl_eval.erl:744: :erl_eval.do_apply/7
    (elixir 1.14.2) src/elixir.erl:294: :elixir.eval_forms/4
```

**Comment by josevalim** [comment]
I think you should be able to remove the Nx dependency. Instead, only list Axon and potentially EXLA. PRs are welcome!

**Comment by seanmor5** [comment]
Fixed in #500
